README.md

```diff
@@ -103,6 +103,7 @@ A non-exhaustive collection of third-party clients and mods for Discord.
 | [RyU](https://github.com/Muunatic/RyU) | Powerful Discord Client written in JavaScript. Lightweight, Efficient, Feature-rich. | [![JavaScript][JavaScript-Badge]][JavaScript-Url] | 🟢 Active |
 | [Swiftcord](https://github.com/cryptoAlgorithm/Swiftcord) | A completely native Discord client for macOS built 100% in Swift and SwiftUI! | [![Swift][Swift-Badge]][Swift-Url] | 🟢 Active |
 | [WebCord](https://github.com/SpacingBat3/WebCord) | A Discord and Fosscord API-less client made with Electron | [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🔵 Active *(Variable[^2]\)* |
+| [Spacebar Chat](https://spacebar.chat/) | Almost fully featured third party Discord API reimplementation, the client | [![JavaScript][JavaScript-Badge]][JavaScript-Url] [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🟢 Active |
```

Comment:
```suggestion
| [Spacebar Chat](https://spacebar.chat/) | Open source, themeable and extendable discord-compatible native Spacebar client | [![TypeScript][TypeScript-Badge]][TypeScript-Url] | 🟢 Active |
```

Spacebar's client repo had its description changed, and they now mostly uses Typescript so the Javascript badge can be removed.
![image](https://user-images.githubusercontent.com/40632486/229054479-4cdaf1ca-8220-4190-8ca5-70a83c6670c6.png)